app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -162,11 +163,12 @@ class AttendeeViewModel(
 
     private fun createAttendee(attendee: Attendee, totalAttendee: Int) {
         if (attendee.email.isNullOrEmpty() || attendee.firstname.isNullOrEmpty() || attendee.lastname.isNullOrEmpty()) {
-            mutableMessage.value = "Please fill in all the fields"
+            detailsNotFilledCount++
+            if (detailsNotFilledCount ==1)
+                mutableMessage.value = "Please fill in all the fields"
```

Comment:
Why?

Reply:
Added this so that the user gets Snackbar message only once, instead of for all the attendees. Right now the user gets Snackbar for all those attendees whose details aren't filled. @iamareebjamal

Reply:
Then just don't set the message value in this method

Reply:
@iamareebjamal Then where should I add the message value ?
Can I run a loop in createAttendees method to check if all the details are filled, and then only call createAttendee method ?
@nikit19 I will fix the travis error.

Reply:
Yes, add a check in AttendeeViewModel only